Header header
float32 drone_timeuint16 tag
uint16 size
float32 angular_rates_trim_r
float32 euler_angles_trim_theta
float32 euler_angles_trim_phi
